dataland-frontend/src/components/resources/frameworkDataSearch/ThreeLayerDataTable.vue

```diff
@@ -0,0 +1,326 @@
+<template>
```

Comment:
Headers of List of Products are wrong:

![image](https://github.com/d-fine/Dataland/assets/35459207/660a255e-b87b-424a-993f-4d722ebf748f)

Reply:
Also this should be formatted as a percentage number i.e. we have to append a % symbol after 30 and 40. If we don't have this kind of formatting yet we should introduce it as a general feature.

Reply:
Fixed